packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/Topics.tsx

```diff
@@ -1,7 +1,74 @@
-import React from 'react';
+import { useCommonNavigate } from 'client/scripts/navigation/helpers';
+import { CWText } from 'client/scripts/views/components/component_kit/cw_text';
+import {
+  CWTab,
+  CWTabsRow,
+} from 'client/scripts/views/components/component_kit/new_designs/CWTabs';
+import React, { useState } from 'react';
+import CreateCommunityHint from '../../CreateCommunity/components/CreateCommunityHint';
+import CreateTopicSection from './CreateTopicsSection';
+import ManageTopicsSection from './ManageTopicsSection';
+import './Topics.scss';
 
-const Topics = () => {
-  return <div className="Topics">Topics</div>;
+export const Topics = () => {
+  const TABS = [
+    { value: 'create-topic', label: 'Create Topic' },
+    { value: 'manage-topics', label: 'Manage Topics' },
+  ];
```

Comment:
this can be defined outside of the component

Reply:
Done!